### PR TITLE
directory-menu@torchipeppo - add new  es.po

### PR DESCRIPTION
**Pull request by haggen88**

### CODE REVIEW

File: directory-menu@torchipeppo/files/directory-menu@torchipeppo/po/es.po [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```
Comment (rcalixte) [inline directory-menu@torchipeppo/files/directory-menu@torchipeppo/po/es.po:6]: It doesn't like this line.

Comment (haggen88) [reply directory-menu@torchipeppo/files/directory-menu@torchipeppo/po/es.po:6]: I am using updated poedit (flatpak), I don't know why there are now conflicts in the .po files

Comment (rcalixte) [reply directory-menu@torchipeppo/files/directory-menu@torchipeppo/po/es.po:6]: Same. I've seen a few of these today so I'm wondering if it's the template for this applet or something else.

Comment (rcalixte) [reply directory-menu@torchipeppo/files/directory-menu@torchipeppo/po/es.po:6]: When I run the script locally, it's fine too. Something changed on the cloud container I think. I'll take a look later on but we can push the fixed commit for now.

**Review by rcalixte — APPROVED** [review]

**Pull request merged**